language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/LimitClauseNodeContext.java

```diff
@@ -74,4 +77,18 @@ public boolean onPreValidation(BallerinaCompletionContext context, LimitClauseNo
     protected Optional<Node> getLastNodeOfClause(LimitClauseNode node) {
         return Optional.of(node.expression());
     }
+
+    @Override
+    public void sort(BallerinaCompletionContext context, LimitClauseNode node, List<LSCompletionItem> completionItems) {
+        Optional<SemanticModel> semanticModel = context.currentSemanticModel();
+        if (semanticModel.isEmpty()) {
+            super.sort(context, node, completionItems);
+            return;
+        }
+        TypeSymbol typeSymbol = semanticModel.get().types().INT;
```

Comment:
We can use more descriptive variable name like `intTypeSymbol` WDYT?

Reply:
+1